tests/model_registry/model_catalog/test_filter_options_endpoint.py

```
@@ -0,0 +1,133 @@
+            elif prop_data.get("type") in ["number", "numeric", "float", "integer", "int"] and "range" in prop_data:
+                range_obj = prop_data["range"]
+                LOGGER.info(
+                    f"Numeric property '{prop_name}' validated: {range_obj.get('min')} to {range_obj.get('max')}"
+                )
+
+        LOGGER.info("All filter options validation passed successfully")
+
+    @pytest.mark.skip(reason="TODO: Implement after investigating backend DB queries")
+    def test_comprehensive_coverage_against_database(
+        self: Self,
+        model_catalog_rest_url: list[str],
```

Ruff (0.14.0)

Unused method argument: `model_catalog_rest_url`

(ARG002)